packages/components/tag/src/tag-body.tsx

```diff
@@ -72,7 +72,10 @@ const getContentWrapperStyles = (props: TTagBodyProps) => {
 };
 
 const TagBody = (props: TTagBodyProps) => {
+  const currentTheme = useTheme();
   const textTone = props.isDisabled ? 'secondary' : undefined;
+  const TextComponent = currentTheme === 'default' ? Text.Detail : Text.Body;
```

Comment:
We need to use a different font-size per theme so we read current theme to calculate which `Text` component we must use.

Reply:
Maybe leave a comment that this would be removed at some point when we make the new theme the default one?